### PR TITLE
Continous Integration with Travis CI. 

### PR DESCRIPTION
**Pull request by arpras**
For now leaving out the deployer modules since passwordless ssh doesn't seem to work on travis. Sent a email to travis support asking for more details on this.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
```
Comment (jdehrlich) [inline .travis.yml]: add the apache license and copyright info

File: test/test_test_utils.py [hunk 2]
```diff
@@ -46,11 +45,7 @@ def test_get_log_for_test(self):
     with open(os.path.join(output_path, 'test.log'), 'w') as f:
       f.write('23:59:59 [main] INFO  Testing_Logs')      
     with open(os.path.join(output_path, 'test.log'), 'w') as f:
-      f.write('23:59:59 [main] INFO  TestClientService - Sent 100')
-
-    log_file = os.path.join(output_path, "test.log")
```
Comment (jdehrlich) [inline test/test_test_utils.py]: are these changes related to travis?

Comment (arpras) [inline test/test_test_utils.py]: Not related to travis.. This was a new test added for coverage and had some issues in it.
